### ISSUE TITLE
Implement new probe to fetch disk size

### ISSUE THREAD

**Issue by qiell** [openebs/node-disk-manager#83]
For AWS EBS volumes we are not getting the capacity of the disk from udev probe. Implement one new probe to fetch the capacity of the disk. This probe will read from system files to fetch capacity of a disk. #72

**Comment by shovanmaity** [comment]
To Implement a new probe follow this pr- #73
steps - 
```
1. add one probe inside pkg/probe
2. provide probe name, priority.
3. write logic to add value in `diskInfo struct`
4. register your probe
```

**Comment by akhilerm** [comment]
Capacity probe implemented in #145
